patterns/1-initial/explicit-shared-ownership.md

```diff
@@ -0,0 +1,47 @@
+# Title
+
+Explicit shared ownership
+
+# Patlet
+
+In a software system there is a component that several teams depend on. Initially developed by a handful of people it has long outgrown it's orignal use-case. Original authors are no longer capable of taking full ownership of that component. Development happens organically. Often there is confusion as to who should make changes, who needs to approve changes. Making ownership explicit, sharing that ownership across dependent teams, making expected behaviour explicit helps remove ambiguity. Writing those learnings down means that it can be changed according to the same process that is used to change the software.
+
+# Problem
+
+An organisation is already using InnerSource best practices in several teams. The architecture of the software offered has grown organically. Talking about code ownership and accountability teams notice that there is a component that is in a worrying state: Developed by an original team of authors it has grown it's userbase way beyond what the original team can handle. From time to time others step up to help out, however there is no formal process established. As a result conflicts arise around who should do the work, who should decide on project direction. Simply forking the component would lead to a lot of duplication and wasted effort. As a result teams are looking for a less intrusive solution to the issue.
+
+# Context
+
+Teams are working in independently but providing one common platform as a service.
+
+The software they are working on has been evolving for a long time. Engineers working on sub-modules have changed over time.
+
+One component is in widespread use but ownership is unclear.
+
+# Forces
+
+Ownership of one component is unclear.
```

Comment:
I'm not against moving it only to the forces section. I lack the deep understanding of the pattern format to make the final call here.

Reply:
After reading this again, I think we can make it more clear by rewording the bullet under Context, as there the important piece is that the component is in widespread use, rather than that ownership is unclear (as the latter is the general problem that this pattern addresses.).

Will suggest a change in a separate thread.